apps/hash-frontend/src/shared/layout/layout-with-header/search-bar.tsx

```diff
@@ -120,10 +131,31 @@ const EntityResult: FunctionComponent<{
         )}`}
       >
         {generateEntityLabel(subgraph, entity)}
+        {entityType && (
+          <Chip
+            color="teal"
+            label={entityType.schema.title}
+            sx={{ cursor: "pointer", ml: 1 }}
+          />
+        )}
+      </Link>
+    </ResultItem>
+  );
+};
+
+const EntityTypeResult: FunctionComponent<{
+  entityType: EntityType;
+}> = ({ entityType }) => {
+  return (
+    <ResultItem>
+      <Link noLinkStyle href={generateLinkParameters(entityType.$id)}>
+        {entityType.title}
+        <Chip
+          color="aqua"
+          label="Entity Type"
+          sx={{ cursor: "pointer", ml: 1 }}
+        />
       </Link>
-      {entityType && (
-        <Chip color="teal" label={entityType.schema.title} sx={{ ml: 1 }} />
-      )}
     </ResultItem>
```

Comment:
Some thoughts I had whilst testing this locally:

1. The color of the "Entity Type" copy is different to the color of the copy in the entity chips
2. Only the text itself is clickable, rather than the entire list item. We probably want the `Link` to wrap the list item in `ResultItem`, rather than be nested inside it
3. When hovering over the chip, the cursor changes from being clickable to non-clickable. We probably need to override how the cursor is being set there
4. When clicking a result, the search bar stays open. We probably want it to close, but maybe leave the search copy populated so the user can easily return to the search results


https://github.com/hashintel/hash/assets/42802102/3a3174d5-d6a4-4dbd-8d83-1fd2fd9deee6

Reply:
Thank you, addressed in https://github.com/hashintel/hash/pull/4267/commits/ac8114f81bde6b2043f3434c58a922fb8cd94013

Apart from the chip colour - I don't want to mess with them as they're also used elsewhere. Can have a proper design for this later